src/main/webapp/new_simulation.html

```diff
@@ -22,6 +22,10 @@ <h1>Create and Run a New BLETH Simulation</h1>
     
     <!-- A form for a new simulation initialization. -->
     <form onload="setAllDynamicUpperBounds()" onchange="setAllDynamicUpperBounds()" id="newSimulationForm">
+        <!-- Description of the simulation. -->
+        <label for="simulationDescription">A short description of the simulation</label><br>
+        <input type="text" id="simulationDescription"><br>
+
```

Comment:
Is this `description` might be relatively long? If so, we might want this to be a `textarea` tag (and if we do, its value should be manually added to the POST request parameters at `new_simulation.js`).

In any case, I think we should also set a `maxlength` attribute.